src/test/java/picard/TestDataProviders.java

```diff
@@ -21,21 +26,46 @@
 public class TestDataProviders {
 
     @Test
-    public void testAllDataProviders() throws IllegalAccessException, InstantiationException, InvocationTargetException {
-        int i = 0;
+    public void IndependentTestOfDataProviderTest() throws IllegalAccessException, InvocationTargetException, InstantiationException {
```

Comment:
When writing tests it's OK (IMO) to use `throws Exception` since we don't to bother updating our tests every time the list of thrown exceptions changes.

Reply:
do you prefer it with throws Exception?

Reply:
Yes, for test methods only